### ISSUE TITLE
Expandable on Vitejs react is not working

### ISSUE THREAD

**Issue by 0xbeny** [lifinance/widget#184]
When I use `path="/swap/*"` for react-router-dom, the routes box will not show.

``` 
<Routes>
          <Route path="/swap/*" element={<Swap />} />
 </Routes>
```

<img width="1171" alt="Screenshot 2024-02-09 at 3 04 54 AM" src="https://github.com/lifinance/widget/assets/55846654/85a5bf62-4509-4aed-b1fd-48cf25ca1fe2">

and when change to 
```
<Routes>
          <Route path="/*" element={<Swap />} />
 </Routes>
```
It'll be working
<img width="1196" alt="Screenshot 2024-02-09 at 3 08 42 AM" src="https://github.com/lifinance/widget/assets/55846654/51f3c04f-e6c6-424c-8dc2-1e6c56d530a5">

Is there any possible fix for that to be able to have LIFI Widget inside the subpages?

**Comment by chybisov** [comment]
@0xbeny thank you! There was indeed an issue and it should fixed in v2.10.1. Please upgrade 🙂

**Comment by 0xbeny** [comment]
> @0xbeny thank you! There was indeed an issue and it should fixed in v2.10.1. Please upgrade 🙂

Thanks for the update! The issue has been fixed.
